CONTRIBUTING

```diff
@@ -0,0 +1,22 @@
+# Contributing to Anon
```

Comment:
Might be best to add `.md` so GitHub formats correctly.

Reply:
sure, I forgot, thanks :)